app/pages/admin.js

```diff
@@ -3,7 +3,7 @@ import Link from "next/link";
 
 export default function Admin() {
 
-  const [links, setLinks] = useState({});
+  const [links, setLinks] = useState({ _id: '', linkObj:{name: '', link: ''}});
```

Comment:
was reading as undefined below, maybe use linkObj?.name below to avoid undefined error.